samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -28,7 +28,8 @@ static struct audio_sink {
 } sinks[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT];
 static struct bt_bap_ep *sources[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
 NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT,
-			  CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE, 8, NULL);
+			  CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE,
```

Comment:
Outside the scope of this PR, but as far as I can tell, `BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU)` and `CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE` is the same, and should probably be updated to use `BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU)` everywhere :)

Reply:
I only found two, if you know of other places, I can change them too.